doc/radosgw/lua-scripting.rst

```diff
@@ -295,6 +300,23 @@ Operations Log
 ~~~~~~~~~~~~~~
 The ``Request.Log()`` function prints the requests into the operations log. This function has no parameters. It returns 0 for success and an error code if it fails.
 
+Background Context
+--------------------
+The background context may be used for various need such as analytics, monitoring and
+caching data from other context executions.
+
+The ``RGW`` Lua table which is accesible from any context saves the data written into it
+while execution and this data could be read and used later in any other execution.
+
+- Background script execution defualt interval is 5 seconds.
+
+- Each RGW instance has its own ``RGW`` Lua table, while the Background Context script
+  will run on every instance.
+
+- The maximum number of entries in the table is 100,000. Each entry has a key and string value
+  of no more than 1KB (together). Lua script will abort with an error if the
+  number of entries or entry size exceeds their limits.
```

Comment:
where does this limit come from? omap?

Reply:
defined and verified in `rgw_lua_utils`. [[1]](https://github.com/ceph/ceph/pull/42779/files#diff-02e90192b6509977587d9358eea0e2d643f31c956c017a84e70705dd3ec7cae0R46)